x/ibc-rs/src/ics02_client/params.rs

```diff
@@ -73,7 +73,7 @@ impl<PSK: ParamsSubspaceKey> ClientParamsKeeper<PSK> {
     ) -> ClientParams {
         let store = infallible_subspace(ctx, &self.params_subspace_key);
 
-        store.params().unwrap() // TODO: Add default
```

Comment:
The TODO seemd to be giving some guidance which might be useful

Reply:
This code needs to removed. Trait impl for params contains blanket impl